examples/keras_spark3_rossmann.py

```diff
@@ -543,8 +549,35 @@ def train_fn(model_bytes):
 conf = SparkConf().setAppName('prediction') \
     .setExecutorEnv('LD_LIBRARY_PATH', os.environ.get('LD_LIBRARY_PATH')) \
     .setExecutorEnv('PATH', os.environ.get('PATH'))
-if LIGHT_PROCESSING_CLUSTER:
-    conf.setMaster(LIGHT_PROCESSING_CLUSTER)
+
+if GPU_INFERENCE_ENABLED:
+    if GPU_INFERENCE_CLUSTER:
+        conf.setMaster(GPU_INFERENCE_CLUSTER)
+    # This config will change depending on your cluster setup.
```

Comment:
Since this is copy-pasted from above, we could move it into a helper function: `set_gpu_conf(conf) -> conf`

Reply:
Thanks for reviewing.

Updated the code. Please help review again, thanks.